arches/app/views/tile.py

```diff
@@ -156,6 +156,7 @@ def post(self, request):
                 if tile_id is not None and tile_id != "":
                     try:
                         old_tile = Tile.objects.get(pk=tile_id)
+                        data["_original_data"] = old_tile.data
```

Comment:
My concern with this change is that it isn't used anywhere within core. We avoid changes to core that support external applications, but aren't clearly part of the API because they can make Arches difficult to maintain. Anyone who wants to modify this post method could easily change or remove this line because it would appear to be cruft. On the other hand, if the outside application stops using it, that change wouldn't be apparent within arches, and it would indeed be cruft at that point. We don't want a situation where developers are afraid to make changes to Arches because they could unintentionally break applications that may or may not be using it.

Reply:
My recollection from Slack/standup is that the use case is subclassing the view. Ultimately decomposing this view might make it easier to subclass, but until then, this seems like a noninvasive change to me.

We could just document it with a single test case and then we'd know not to break it (or we could choose to break it if/only if decomposing the view to encourage easier subclassing.)

Reply:
I have to agree with @chiatt that adding this, even if it's just one line, sets a dangerous precedent.  I know that there are other developers out there that would point to this and say "well since you allowed this here, can you allow my edit?"  I don't want to be super dogmatic, but this does feel like a slippery slope.  Even if we had a test and documented it, I would still think it's not a great idea.  

Now onto an alternate solution.
If all we want to do is check the previous revision of a tile on save in a signal, you could easily query the edit_log for that information.
```
SELECT *
FROM public.edit_log
WHERE tileinstanceid = 'c85dbd19-9697-4cb7-8997-2c72f0ff3132'
ORDER BY timestamp DESC
LIMIT 2
```
This would give you the current edit and the previous edit.

I do realize that this is an additional db query which is not usually preferred, but I was able to test this against a very large instance of arches that contains 40M edit_log entries.  After adding an index to the `tileinstanceid` column (which you have to have), it took milliseconds to execute that query.

Reply:
> I have to agree with @chiatt that adding this, even if it's just one line, sets a dangerous precedent. I know that there are other developers out there that would point to this and say "well since you allowed this here, can you allow my edit?" I don't want to be super dogmatic, but this does feel like a slippery slope. Even if we had a test and documented it, I would still think it's not a great idea.
> 
> Now onto an alternate solution. If all we want to do is check the previous revision of a tile on save in a signal, you could easily query the edit_log for that information.
> 
> ```
> SELECT *
> FROM public.edit_log
> WHERE tileinstanceid = 'c85dbd19-9697-4cb7-8997-2c72f0ff3132'
> ORDER BY timestamp DESC
> LIMIT 2
> ```
> 
> This would give you the current edit and the previous edit.
> 
> I do realize that this is an additional db query which is not usually preferred, but I was able to test this against a very large instance of arches that contains 40M edit_log entries. After adding an index to the `tileinstanceid` column (which you have to have), it took milliseconds to execute that query.

I'm going to close this PR given @apeters comments and proposed solution.